app/src/main/java/infinite_zoomer/model/Stroke.java

```diff
@@ -108,4 +114,8 @@ protected void updateBoundingCircle() {
             }
         }
     }
+
+    public List<Line> getmLines() {
```

Comment:
```suggestion
    public List<Line> getLines() {
```

The `m` in `mLines` just stands for `member variable` `lines`. It's just something I like doing to make it clear that `lines` is a private field of `Stroke`. ☺